BREAKING_CHANGES.md

```diff
@@ -1,3 +1,15 @@
+2023-04-07 7d70009
+
+zfs_datasets are now split into zfs_fs and zfs_volume. zfs_type got removed. size is only available for zfs_volume
```

Comment:
```suggestion
Date: 2023-04-07

Commit: 7d70009

Changes:

    ZFS datasets have been split into two types: zfs_fs and zfs_volume.
    The zfs_type attribute has been removed.
    The size attribute is now only available for zfs_volume.

Updated example/zfs.nix file:

```bash
datasets = {
  zfs_fs = {
    type = "zfs_fs";
    mountpoint = "/zfs_fs";
    options."com.sun:auto-snapshot" = "true";
  };
  zfs_unmounted_fs = {
    type = "zfs_fs";
    options.mountpoint = "none";
  };
  zfs_legacy_fs = {
    type = "zfs_fs";
    options.mountpoint = "legacy";
    mountpoint = "/zfs_legacy_fs";
  };
  zfs_testvolume = {
    type = "zfs_volume";
    size = "10M";
    content = {
      type = "filesystem";
      ...

Note: The zfs_type attribute has been replaced with a type attribute for each dataset, and the size attribute is only available for zfs_volume. These changes have been reflected in the example/zfs.nix file.
```

```